userguide/.gitignore

```diff
@@ -2,3 +2,5 @@
 .hugo_build.lock
 /public
 resources/
+# Themes
+themes/
```

Comment:
Why this addition? The User Guide build doesn't involve the use of anything under `./themes`.

Reply:
You are right, this entry is not strictly necessary, so I removed it.